src/internal/cluster/zarf.go

```diff
@@ -99,24 +98,129 @@ func (c *Cluster) StripZarfLabelsAndSecretsFromNamespaces() {
 	spinner.Success()
 }
 
+// PackageSecretNeedsWait checks if a package component has a running webhook that needs to be waited on.
+func (c *Cluster) PackageSecretNeedsWait(deployedPackage types.DeployedPackage, component types.ZarfComponent, skipWebhooks bool) (needsWait bool, waitSeconds int, hookName string, err error) {
```

Comment:
```suggestion
func (c *Cluster) PackageSecretNeedsWait(deployedPackage types.DeployedPackage, component types.ZarfComponent, skipWebhooks bool) (needsWait bool, waitSeconds int, hookName string) {
```
err is no longer needed

Reply:
Removed err return value